src/include/uapi/ipc/xtensa.h

```diff
@@ -39,12 +39,25 @@
 #ifndef __INCLUDE_UAPI_IPC_XTENSA_H__
 #define __INCLUDE_UAPI_IPC_XTENSA_H__
 
+#include <arch/offset-defs.h>
+
 /*
  * Architecture specific debug
  */
 
+struct sof_ipc_dsp_oops_header {
+	uint32_t arch;		/* Identifier of architecture */
+	uint32_t configidhi;	/* ConfigID hi 32bits */
+	uint32_t configidlo;	/* ConfigID lo 32bits */
+	uint32_t totalsize;	/* Total size of oops message */
+	uint32_t stackoffset;	/* Offset to stack pointer from beginning of
+				 * oops message
+				 */
+} __attribute__((packed));
+
 /* Xtensa Firmware Oops data */
 struct sof_ipc_dsp_oops_xtensa {
+	struct sof_ipc_dsp_oops_header hdr;
```

Comment:
I'll have to update https://github.com/thesofproject/sof/blob/master/tools/coredumper/sof-coredump-reader.py .
@lgirdwood @akloniex you think in separate PR, or inside this one?